.homeycompose/capabilities/operation_mode_state.json

```diff
@@ -14,49 +17,70 @@
       "id": "idle",
       "title": {
         "en": "Idle",
-        "fr": "Inactif"
+        "fr": "Inactif",
+        "nl": "Inactief",
+        "sv": "Inaktiv",
+        "es": "Inactivo"
       }
     },
     {
       "id": "dhw",
       "title": {
-        "en": "Domestic house water",
-        "fr": "Eau chaude sanitaire"
+        "en": "Domestic hot water",
+        "fr": "Eau chaude sanitaire",
+        "nl": "Sanitair warm water",
+        "sv": "Varmvattenladdning",
```

Comment:
"sv": "Varmvattenberedning".

Reply:
I've seen that both terms exist, I will keep varmvattenladdning which seems more proper to heat pumps (https://www.thermia.se/varmepump-kunskap/hur-fungerar-en-varmepump/varmepump-teknik/variabel-varmvattenladdning/)